keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -452,94 +457,192 @@ change are understandable. This may include API specs (though not always
- - this pod must run on a windows host, and kubelets must reject it if not on windows hosts
- - all pods marked privileged on windows must have host network enabled, if not the pod does not validate
+##### Alternatives
 
-In beta there is a possibility to enable the privileged container to be a part of a different network component.  If this feature is enabled we will use the existing Pod HostNetwork field to enable/disable.
+Option 1: Re-use `SecurityContext.Privileged` field.
 
-#### <b> ContainerD Support Only</b>
-There are no plans to update Docker to have support for Privileged containers due to requirements for support HCSv2.  The additional fields for docker would be ignored. The default value should be false.
+Re-using the existing `SecurityContext.Privileged` field was considering and here are the pros/cons considered:
 
-Need to investigate if the choice of base image would cause errors when pulling with docker.
+Pros
 
-#### <b> Feature Gates</b>
-Although there are no changes required to the kubernetes API, the ability to pass the the privileged flag to the CRI should be behind a feature gate in Kubelet:  
+- The field already exists and many policy tools already leverage it.
 
-https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-stages
+Cons
 
+- Privileged containers on Windows will operate very differently than privileged containers on Linux. Having a new field should help avoid confusion around the differences between the two.
+- The privileged field does not have clear meaning for Linux containers today (see comments above).
+- `WindowsSecurityContextOptions.RunAsUserName` will the the primary way of restricting access to host/node resources (See [Container users](#container-users)). It is desirable that `RunAsUserName` and `HostProcess` fields live on the same property.
+- API validation to ensure all containers are either privileged or not will be difficult because there is no way of definitively knowing that a pod is intended for a Windows node.
 
-### Test Plan
+#### Host Network Mode
+
+Host Network mode for privileged Windows containers will always be enabled, as the pod will automatically get the host IP.
+
+Privileged Windows containers will be unable to align to pod namespaces due to limitations in the Windows OS. This functionality will likely be enabled in the future through a new KEP.
+
+Because of this we will require that `hostNetwork` is set to `true` when scheduling privileged pods. This will allow existing policy tools to detect and act on privileged Windows containers without any updates. In the future if/when functionality is added to support joining privileged containers to pod networks this validation will be revisited.
```

Comment:
Because there is currently unpredictable behavior if hostNetwork is set today (https://github.com/kubernetes/enhancements/pull/2288#discussion_r572377500) I didn't want to add validation that could potentially break current user-workflows.

Reply:
Sorry - I misunderstood your comment.

Yes, we I plan add checks at creation time in apiserver.
I added notes on a additional kubelet check based on https://github.com/kubernetes/enhancements/pull/2288#discussion_r564935794